src/apps/explorer/pages/Home.tsx

```diff
@@ -11,11 +13,30 @@ const Wrapper = styled.div`
   top: var(--height-bar-default);
 `
 
-export const Home: React.FC = () => (
-  <Wrapper>
-    <h1>Gnosis Protocol Explorer</h1>
-    <p>Welcome :)</p>
-  </Wrapper>
-)
+export const Home: React.FC = () => {
+  const [fee, setFee] = useState('')
+  const [trades, setTrades] = useState<string[]>([])
+
+  useEffect(() => {
+    getFeeQuote(Network.Mainnet, '0x6810e776880c02933d47db1b9fc05908e5386b96')
```

Comment:
U can kill this if u want. It was just to play around

Reply:
Ok, will remove.